examples/KotlinExample/dev.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
could be another just file? or it it not recommended to have multiple just files

Reply:
yeah nice, done